pkg/preparer/orchestrate.go

```diff
@@ -94,6 +98,7 @@ func (p *Preparer) WatchForPodManifestsForNode(quitAndAck chan struct{}) {
 				go p.handlePods(podChanMap[podId], quitChanMap[podId])
 			}
 			podChanMap[podId] <- result.Manifest
+			errs <- nil
```

Comment:
Have you tested this? I am pretty sure the `waitForTermination` method will hang forever in main because it sends a value on the `err` channel, but will never be received if no `StatusPort` was configured.